be/src/exec/pipeline/exchange/sink_buffer.cpp

```diff
@@ -229,6 +246,7 @@ void SinkBuffer::_try_to_send_rpc(const TUniqueId& instance_id) {
             } else {
                 std::lock_guard<std::mutex> l(*_mutexes[ctx.instance_id.lo]);
                 _process_send_window(ctx.instance_id, ctx.sequence);
+                _update_network_overhead(ctx.instance_id, send_timestamp, result.receive_timestamp());
```

Comment:
should check receive_timestamp is set

Reply:
Is it necessary? Because it has no side effect